docker/Dockerfile.pgi

```diff
@@ -0,0 +1,108 @@
+# Installing PGI needs to be done interactively to accept the license. The
+# installation of the compiler and MPI is done using the default options.
+# During the installation the PGI compiler, multiple versions of the CUDA
+# toolkit. This unnecessarily increases the size of the image, so before the
```

Comment:
toolkit are getting installed?

Reply:
Yes, 8, 9, and 10 get installed twice because they have two backends for their compilers. So you end up with 6 cuda toolkits in the image.

Reply:
I meant, modify the text to read "toolkit are getting installed" :) What are 8, 9, 10?

Reply:
CUDA 8.0, CUDA 9.2, and CUDA 10.1 are installed

Reply:
What image are you starting from?  Ubuntu 18.04?